parsons/mailchimp/mailchimp.py

```diff
@@ -0,0 +1,391 @@
+        that argument descriptions here are sourced from Mailchimp's official
+        API documentation.
+
+        `Args:`
+            fields: list of strings
+                A comma-separated list of fields to return. Reference
+                parameters of sub-objects with dot notation.
+            exclude_fields: list of strings
+                A comma-separated list of fields to exclude. Reference
+                parameters of sub-objects with dot notation.
+            count: int
+                The number of records to return. Default value is 10. Maximum
+                value is 1000.
+            offset: int
+                The number of records from a collection to skip. Iterating over
+                large collections with this parameter can be slow. Default
+                value is 0.
+            before_date_created: string
+                Restrict response to lists created before the set date. We
+                recommend ISO 8601 time format: 2015-10-21T15:41:36+00:00.
+            since_date_created: string
+                Restrict results to lists created after the set date. We
+                recommend ISO 8601 time format: 2015-10-21T15:41:36+00:00.
+            before_campaign_last_sent: string
+                Restrict results to lists created before the last campaign send
+                date. We recommend ISO 8601 time format:
+                2015-10-21T15:41:36+00:00.
+            since_campaign_last_sent: string
+                Restrict results to lists created after the last campaign send
+                date. We recommend ISO 8601 time format:
+                2015-10-21T15:41:36+00:00.
+            email: string
+                Restrict results to lists that include a specific subscriber's
+                email address.
+            sort_field: string, can only be 'date_created' or None
+                Returns files sorted by the specified field.
+            sort_dir: string, can only be 'ASC', 'DESC', or None
+                Determines the order direction for sorted results.
+
+        `Returns:`
+            Table Class
+        """
+        params = {'fields': fields,
+                  'exclude_fields': exclude_fields,
+                  'count': count,
+                  'offset': offset,
+                  'before_date_created': before_date_created,
+                  'since_date_created': since_date_created,
+                  'before_campaign_last_sent': before_campaign_last_sent,
+                  'since_campaign_last_sent': since_campaign_last_sent,
+                  'email': email,
+                  'sort_field': sort_field,
+                  'sort_dir': sort_dir}
+
+        response = self.get_request('lists', params=params)
+        tbl = Table(response['lists'])
+        logger.info(f'Found {tbl.num_rows} lists.')
+        if tbl.num_rows > 0:
+            return tbl
+        else:
+            return None
```

Comment:
This method (and those below) should return an empty table instead of `None`.

Reply:
Fixed in d9e1550.